MachObfuscator/Mach/Mach+Replacing.swift

```diff
@@ -16,6 +32,11 @@ private extension Mach {
             data.replaceStrings(inRange: classNameSection.range.intRange, withMapping: map.classNames)
         }
 
+        if let methTypeSection = objcMethTypeSection {
```

Comment:
What do you think about running this code block only if `erase-methtype` has not been provided? This could save some time.

Reply:
`erase-methtype` is dangerous and IMO should rarely (if ever apart from experiments) be used, so the CPU savings are not so important. Other checks in replaceSymbols are not controlled by flags, so adding one of `methTypeSection` would be a bit inconsistent and would complicate parameters of `replaceSymbols`

Reply:
Ok, if it ever becomes an issue - we can change it later.